eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -42,7 +42,7 @@ extends:
           - name: PythonVersion
             value: '3.13'
           - name: SpecGenSdkVersion
-            value: 'latest'
+            value: '0.2.2'
```

Comment:
Proper fix is to use a package-lock.json to pin the package and all its transitive deps.  Can we do this instead?

Reply:
@mikeharder I added `spec-gen-sdk` to dev dependencies of root 'package.json'.